packages/frontend/components/Borrow/Overview.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from "react"
+import { useTheme } from "@mui/material/styles"
+import {
+  Box,
+  Card,
+  CardContent,
+  Collapse,
+  Container,
+  Divider,
+  Grid,
+  Tooltip,
+  Typography,
+} from "@mui/material"
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined"
+import AddCircleIcon from "@mui/icons-material/AddCircle"
+import CancelIcon from "@mui/icons-material/Cancel"
+
+import CurrencyCard from "./CurrencyCard"
+import LTVProgressBar from "./LTVProgressBar"
+import Image from "next/image"
+import PreviewTransaction from "./PreviewTransaction"
+import ClickableTooltip from "../Layout/ClickableTooltip"
+
+export default function Overview() {
+  const { palette } = useTheme()
+  const [showProvider, setShowProvider] = useState(false)
+
+  return (
+    <Container
```

Comment:
Container should only be used as a root of the layout, see documentation https://mui.com/material-ui/react-container/#main-content

Here using a `Box` is a better fit